charts/traceability-foss-backend/values.yaml

```diff
@@ -111,58 +115,52 @@ ingress:
   hosts: []
   tls: []
 
-##############################
-# Application Configuration  #
-#############################
-
-springprofile: <springprofile>
-
 healthCheck:
   enabled: true #<healthCheck.enabled>
 
 traceability:
-  bpn: <traceability.bpn>
-  url: <traceability.url>
+  bpn: "CHANGEME" #<traceability.bpn>
+  url: "" #<traceability.url>
 
 datasource:
-  url: <datasource.url>
-  username: <datasource.username>
-  password: <datasource.password>
+  url: jdbc:postgresql://tracex-backend-postgresql:5432/trace
+  username: trace
+  password: "CHANGEME" #<datasource.password>
 
 oauth2:
-  clientId: <oauth2.clientId>
-  clientSecret: <oauth2.clientSecret>
-  clientTokenUri: <oauth2.clientTokenUri>
-  jwkSetUri: <oauth2.jwkSetUri>
-  resourceClient: <oauth2.resourceClient>
+  clientId: "CHANGEME" #<oauth2.clientId>
+  clientSecret: "CHANGEME" #<oauth2.clientSecret>
+  clientTokenUri: "https://changeme.com" #<oauth2.clientTokenUri>
+  jwkSetUri: "https://changeme.com" #<oauth2.jwkSetUri>
```

Comment:
I do not think that this is a good style. A user should know that the uri of clientTokenUri has to be set. Please remove changeme.com